blue_modules/notifications.ts

```
@@ -0,0 +1,636 @@
+/**
+ * Initializes the Notifications module.
+ * This should be called from your app's entry point (e.g., App.tsx) with the appropriate callback.
+ *
+ * @param onProcessNotifications - Callback to process notifications when app is active
+ * @returns {Promise<void>}
+ */
+export const initializeNotifications = async (onProcessNotifications: () => void): Promise<void> => {
+  // Fetching to see if app uses a custom GroundControl server, not the default one
+  try {
+    const baseUriStored = await AsyncStorage.getItem(GROUNDCONTROL_BASE_URI_KEY);
+    if (baseUriStored && baseUriStored.trim()) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)